doc/users_guide.rst

```diff
@@ -1392,9 +1392,10 @@ available:
   solver for all examples. It is strongly recommended to install umfpack
   and its SciPy wrappers to get good performance.
 - 'ls.scipy_iterative': Interface to SciPy iterative solvers.
-- 'ls.pyamg': Interface to PyAMG solvers.
+- 'ls.pyamg': Interface to PyAMG solvers, see `PyAMG Tutorial`_.
 - 'ls.petsc': Interface to Krylov subspace solvers of PETSc supporting parallel
-  use.
+  use. See `PETSc Users Manual`_, Chapter 4, for more details about available
+  Krylov methods and preconditioners.
```

Comment:
What about linking the relevant example(s) here? Not sure - it is a complex topic.